<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelsImportCSV</class>
 <widget class="QDialog" name="ChannelsImportCSV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV file structure</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/icons/icons.qrc">
    <normaloff>:/icons/apps/16/accessories-text-editor.png</normaloff>:/icons/apps/16/accessories-text-editor.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>CSV file structure</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSeparator">
        <property name="text">
         <string>Separator:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="editSeparator">
        <property name="text">
         <string>;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="labelFirstLine">
        <property name="text">
         <string>First line:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="radioFirstHeader">
        <property name="text">
         <string>First line contains column names</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="radioFirstContent">
        <property name="text">
         <string>First line does not contain column names</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelColumnList">
        <property name="text">
         <string>Column list:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QLabel" name="labelColumnListInfo">
        <property name="text">
         <string>&lt;i&gt;Please sort your columns from first to last.&lt;/i&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QListWidget" name="columns">
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::InternalMove</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Number</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Channel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>URL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Categories</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Language</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EPG ID</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editSeparator</tabstop>
  <tabstop>radioFirstHeader</tabstop>
  <tabstop>radioFirstContent</tabstop>
  <tabstop>columns</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChannelsImportCSV</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChannelsImportCSV</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
